### PR TITLE
Homework9: Vladislav Savchuk 

### PR DESCRIPTION
**Pull request by VladislavSavchuk**
# Decisions of tasks homework #9

- Strings with the specified character
- Candles
- Counting the number of letters

### CODE REVIEW

**Comment by VladislavSavchuk** [comment]
Hello! made a review homework #9, please)

File: homework/homework9/candles.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        burned += candles
+        remaining += candles
+        new_candles = remaining // make_new
+        remaining %= make_new
+        return burn_candles(new_candles, burned, remaining)
+
+    return burn_candles(candles_number, 0, 0)
+
+
+assert total_candles(5, 2) == 9
```
Comment (Stml89) [inline homework/homework9/candles.py]: it would be nice to have an error message here

File: homework/homework9/candles.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+assert total_candles(1, 2) == 1
+assert total_candles(15, 5) == 18
+assert total_candles(12, 2) == 23
+assert total_candles(6, 4) == 7
+assert total_candles(13, 5) == 16
+assert total_candles(2, 3) == 2
+
+print("All tests passed!")
```
Comment (Stml89) [inline homework/homework9/candles.py]: no errors means that everything is good. Pls remove this line

File: homework/homework9/find_specified_character.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+"""
+
+
+def find_specific_character(data: str) -> str:
+    """
+    This function finds the “#” character and
+    if this character is found - deletes the previous character
+    from the string
+
+    data: str
```
Comment (Stml89) [inline homework/homework9/find_specified_character.py]: pls, add more information here. 
https://google.github.io/styleguide/pyguide.html#383-functions-and-methods

**Review by Stml89 — APPROVED** [review]
fix small comments and you can merge PR

**Comment by VladislavSavchuk** [comment]
fixed code according to the comments

**Pull request merged**